README.md

```diff
@@ -143,6 +172,17 @@ Using the same front-end application and Rails API from the previous question, h
 
 If the Post creation is successful, the new Post should be printed to the browser console. Otherwise, an error message should be printed to the console.
 
-```js
-// Your answer goes here...
+```
+$.jax({
+  type: 'POST'
+  data: {title: title, body:body}
```

Comment:
`title` and `body` aren't currently defined. So in this case you would just want to use hardcoded strings or define the variables `title` and `body`